langserver/handler.go

```diff
@@ -437,11 +439,24 @@ func (h *langHandler) lint(ctx context.Context, uri DocumentURI) ([]Diagnostic,
 				entry.Filename = filepath.ToSlash(entry.Filename)
 			}
 			word := ""
+
+			// if the linter returns 0 based columns we get a probleme here without the offset
+			// as we subtract one later we add one extra to the offset
+			if config.LintOffsetColumns > 0 {
+				entry.Col = entry.Col + config.LintOffsetColumns + 1
```

Comment:
what's the reason for the `+ 1` here? on my system the pylint is still offset with LintOffsetColumns = 1, since 2 is added to the col, and not possible to just add 1